packages/webapp/__tests__/setup.ts

```diff
@@ -7,20 +7,7 @@ process.env.NEXT_PUBLIC_API_URL = 'http://localhost:3000';
 process.env.NEXT_PUBLIC_WEBAPP_URL = '/';
 
 /* eslint-disable @typescript-eslint/no-explicit-any */
-
-jest.mock('next/dynamic', () => (func: () => Promise<any>) => {
-  let component: any = null;
-  func().then((module: any) => {
-    component = module.default;
```

Comment:
This caused a bazillion errors due to it expecting the dynamic import to be default:

![notAllDynamicAreDefault](https://github.com/user-attachments/assets/551c0bed-4d4d-41f0-b65f-d5c1c425f3cf)

But not all components we want to import dynamically are necessarily exported as a default.

Removing this made all tests pass, except one, which passed when I wrapped it in an act. So it didn't seem to be really necessary?

The code was written 4 years ago, so maybe the recent updates we've done to NextJs made it redundant